exercises/02/readme.md

```diff
@@ -14,6 +14,8 @@ After completing these steps you'll be familiar with how you can use the CDS com
 
 ### 1. Initialize a new CAP project
 
+Before you start creating a new project it's highly recommended to create a dedicated directory for this CodeJam within your users directory instead of any system directories (e.g. DO NOT work in C:\Windows\System32).
```

Comment:
- it's highly recommended to -> we recommend that you
- good tip, btw!

Reply:
done!